src/components/Button/ButtonAddPostTopic.js

```diff
@@ -35,6 +36,10 @@ export default ButtonAddPostTopic;
 
 const styles = StyleSheet.create({
   container: {
+    position: 'absolute',
+    bottom: normalize(30),
+    right: 0,
+    zIndex: 99,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There seems to be a typo in the `backgroundColor` property of the `container` style. It's currently set to `'#0000'`, which is not a valid color code. If you intended to make the background transparent, use `'#00000000'` for full transparency or `'#000000FF'` for black.
```diff
-    backgroundColor: '#0000'
+    backgroundColor: '#00000000'
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->